.github/workflows/ci-lite-test-full.yml

```diff
@@ -0,0 +1,120 @@
+name: Test Lite full
```

Comment:
I don't think this needs to go in bug-fix release.

Reply:
done